Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1070,11 +1070,12 @@ def compile(self, more, haveInstructions, glyfTable):
 				data = data + struct.pack(">HH", self.firstPt, self.secondPt)
 				flags = flags | ARG_1_AND_2_ARE_WORDS
 		else:
+			x, y = self.roundOffset()
```

Comment:
I would think that `self.roundOffset()` would simply round the `self.x` and `self.y` in place without returning a value.
If the method does return a value, I would maybe call it `self.getRoundOffsets()`.
BTW, the `GlyphCoordinates.toInt()` method does not have a return value, for example.

But these are details anyway. Thanks @khaledhosny!